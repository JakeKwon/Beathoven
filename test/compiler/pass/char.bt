// char
char ch_a = 'a';
char ch_b;
ch_b = 'b';
print("ch_a is ",ch_a,"\n");
print("ch_a is ",ch_b);