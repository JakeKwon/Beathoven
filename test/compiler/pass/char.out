ch_a is a
ch_a is b